### PR TITLE
HLM-477 | Added Divoc listener to log responses from registry-async-handler

### PR DESCRIPTION
**Pull request by ajay-beehyv**
@egov-joy @prasanna-egov @saiprakash-v

### CODE REVIEW

**Review by prasanna-egov — APPROVED** [review]

**Review by prasanna-egov — APPROVED** [review]

File: backend/vaccination_api/cmd/analytics_feed/main.go [hunk 1]
```diff
@@ -147,6 +148,20 @@ dt Date
 	if err != nil {
 		log.Fatal(err)
 	}
+	_,err = connect.Exec(`
+	CREATE TABLE IF NOT EXISTS transactionResponseV1 (
```
Comment (egov-joy) [inline backend/vaccination_api/cmd/analytics_feed/main.go:152]: The table name doesn't clearly give away what this table is used for. Can be more descriptive.

File: backend/vaccination_api/cmd/analytics_feed/main.go [hunk 2]
```diff
@@ -232,12 +248,14 @@ func startCertificateEventConsumer(err error, connect *sql.DB, callback MessageC
 			} else {
 				log.Errorf("Error in processing the certificate %+v", err)
 			}
-			services.PublishProcStatus(models.ProcStatus{
-				Date:              time.Now(),
-				PreEnrollmentCode: preEnrollmentCode,
-				ProcType:          procType,
-				Status:            string(status),
-			})
+			if(procType != "procStatus_event"){
```
Comment (egov-joy) [inline backend/vaccination_api/cmd/analytics_feed/main.go:251]: was this change part of this story or something that was added for a different story and missed to be merged? Don't see why this change is needed for this story.

File: backend/vc-certification-service/src/middleware/auth.middleware.js [hunk 3]
```diff
@@ -22,6 +22,6 @@ module.exports = function (req, res, next) {
         next();
     } catch (err) {
         console.error("Error in verifying token: ", err);
-        res.status(401).send({error: "auth failed, check bearer token"});
+        res.status(400).send({error: "auth failed, check bearer token"});
```
Comment (egov-joy) [inline backend/vc-certification-service/src/middleware/auth.middleware.js:25]: Seems unrelated to this story.

File: backend/vaccination_api/cmd/analytics_feed/main.go (hunk 1 above)
Comment (egov-joy) [reply backend/vaccination_api/cmd/analytics_feed/main.go:152]: Should be consistent with the method names used below as well.. i.e. `saveVcTransactionEvent`

File: backend/vaccination_api/cmd/analytics_feed/main.go (hunk 2 above)
Comment (ajay-beehyv) [reply backend/vaccination_api/cmd/analytics_feed/main.go:251]: No, It's an existing mistake from PR #1095, without restricting procType, PublishProcStatus will loop continously

File: backend/vc-certification-service/src/middleware/auth.middleware.js (hunk 3 above)
Comment (ajay-beehyv) [reply backend/vc-certification-service/src/middleware/auth.middleware.js:25]: It's merge conflict, PR #1176 merged before this PR

**Review by egov-joy — APPROVED** [review]
